### PR TITLE
Fixfooter

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by julianecasier — DISMISSED** [review]

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
No review a SM

**Review by julianecasier — APPROVED** [review]

**Review by julianecasier — DISMISSED** [review]

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
possible optimisation with modals

**Review by julianecasier — DISMISSED** [review]

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Resolve a conflict, one modal with props

**Review by julianecasier — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/vite.config.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-const path = require("path");
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 
+const path = require("path");
+
```
Comment (LoicBrassart) [inline frontend/vite.config.js]: What? Why ?

File: frontend/src/components/Footer/index.jsx [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
-        <li>The team</li>
-      </ul>
+      <Modal button="About">
+        <h2>Hello Modal </h2>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus,
+          itaque fuga? Maxime corporis, nemo ipsum sunt porro veritatis soluta
+          illo, aspernatur eligendi ea doloremque velit natus totam? Nihil,
+          facere quae sint porro soluta corporis veritatis tempora obcaecati?
+          Aspernatur, officiis adipisci!
```
Comment (LoicBrassart) [inline frontend/src/components/Footer/index.jsx:14]: Have you asked the client for the final texts ? If that's not the case, he won't give any and you'll stay stuck with filler text

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor fixes

File: frontend/src/components/Footer/index.jsx (hunk 2 above)
Comment (julianecasier) [reply frontend/src/components/Footer/index.jsx:14]: Yes we asked, he told us to put some lorem ipsum waiting for his text.

**Review by julianecasier — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Review by LoicBrassart — DISMISSED** [review]
No more Questions !

**Review by julianecasier — APPROVED** [review]

**Comment by julianecasier** [comment]
Reinjected after the merge (sorry Loic, I accidently added it directly to you instead of Luigi first)

**Review by LoicBrassart — APPROVED** [review]
Fine by me !

**Review by LuigiMenez — APPROVED** [review]
Ok

**Pull request merged**